test/jenkins_tests/multi_node_docker_test.py

```diff
@@ -218,6 +232,8 @@ def start_ray(self,
             self._start_worker_node(docker_image, mem_size, shm_size,
                                     num_cpus[1 + i], num_gpus[1 + i],
                                     development_mode)
+        # Sleep to wait for the registration of raylet workers.
+        time.sleep(5)
```

Comment:
The [failure](https://amplab.cs.berkeley.edu/jenkins/job/Ray-PRB/7010/console) comes from non-registered worker. Since we have add the exception to trigger the retry, this is not necessary, but we can receive some retry message in CI. That is fine.

Reply:
After the test, there is no retrying message. Thanks!